### DISCUSSION TITLE
2.5.11 - not in Arduino yet?

### DISCUSSION THREAD

**Discussion by mechatroniks-git** [SpenceKonde/megaTinyCore#673] [Q&A] [answered]
@SpenceKonde Thanks for the update, but it's not showing as updatable in boards manager. 2.5.10 is latest here on 1.8.19, unless I'm missing something.

**Comment by SpenceKonde** [comment, marked as answer]
Yes I'm working on it right now, I encountered a bunch of problems with DxCore 1.4.10 which is being released at the same time.
